### ISSUE TITLE
no-process-exit throws on process.once (but not on process.on)

### ISSUE THREAD

**Issue by Izhaki** [sindresorhus/eslint-plugin-unicorn#133]
From [the docs](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/48ef6e1de5d92f6538b8399f8f4bfe06a7c3ec8f/docs/rules/no-process-exit.md):

> It also allows process.exit() to be called in process.on('<event>', func) event handlers.

Any reason `process.once` is disallowed?

As in:
```javascript
process.once( 'SIGINT', () => {
    process.exit() // Throws Only use `process.exit()` in CLI apps. Throw an error instead. (unicorn/no-process-exit)
})

process.on( 'SIGINT', () => {
    process.exit() // Doesn't thorw
})
```

**Comment by jamestalmage** [comment]
No good reason. If it's good in `on` it's good in `once`. 
PR Welcome.
Relevant line of code is here: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/rules/no-process-exit.js#L17

**Comment by Izhaki** [comment]
@jamestalmage Done.
